etc/version.lock.json

```diff
@@ -1332,9 +1363,17 @@
     "version": 6
   },
   "68994a6c-c7ba-4e82-b476-26a26877adf6": {
+    "min_stack_version": "8.0",
+    "previous": {
```

Comment:
all of the rules which existed and then had a `min_stack_version` added should indicate that they have been "forked" with this `previous` structure